docs/iidm/exporter/iidm.md

```diff
@@ -12,12 +12,26 @@ For more information about the IIDM model, see [here](../model/index.md).
 IIDM networks can be serialized in XML files. The IIDM exporter generates files with a `*.xiidm` extension.
 
 The IIDM exporter has three exporting modes:
-
    - **First mode**: Exports the network and its extensions in a unique file.
     
    - **Second mode**: Exports the network in a file and the extensions in another file.
     
    - **Third mode**: Exports the network in a file and each extension type in a separate file. 
+ 
+There is another way to export a network, it's a light export method called incremental export.  
+
+That export method, when applied on a network, generates (by default) three files:   
+
+   - `base-STATE.xiidm` file: contains only network elements having state attributes.
+    
+   - `base-TOPO.xiidm` file: contains only network elements having topology attributes.
+   
+   - `base-CONTROL.xiidm` file: contains only network elements having control attributes.
+
+In case you want to update a network, you don't need a whole network, you just need network elements containing useful attributes, that's why that export method exist.
+The exported files, in a later stage,  allow you to update a network control, state or topology elements's attributes separately.
+
+
```

Comment:
A bit difficult to review without proposing a paragraph. I prefer an introduction more simple and less technical like this:

The IIDM exporter has also a lightest mode that allows to export only fields that described the network variables. This mode is more commonly called incremental mode. It exports the network in three files, one dedicated to the state variables, one dedicated to the topology and another one dedicated to control variables. This export is light because its design allows to update control, state or topology elements's attributes of a network in a separate way.

Reply:
*lighter